packages/kolibri-common/components/Cards/AccessibleFolderCard.vue

```diff
@@ -77,26 +77,18 @@
         default: null,
       },
     },
-    computed: {
-      headerStyles() {
-        return {
-          color: this.$themeTokens.text,
-          borderRadius: '4px',
-          height: '24px',
-          width: '74px',
-          margin: '0',
-          backgroundColor: this.$themePalette.grey.v_100,
-        };
-      },
-    },
   };
 
 </script>
 
 
 <style lang="scss" scoped>
 
-  .header-bar {
+  /deep/ .k-horizontal-with-small-thumbnail.k-thumbnail-align-right .k-thumbnail {
+    margin: 0;
```

Comment:
I think this should be fixed in KCard, its a little inconsistent that when we have thumbnailDisplay=small the thumbnail is bigger than when we have thumbnailDisplay=large :sweat_smile:.

| thumbnailDisplay small (+margin: 0) | thumbnailDisplay large |
| --- | --- |
| ![image](https://github.com/user-attachments/assets/28999612-b78e-44ed-a37f-f1816d57e287) | ![image](https://github.com/user-attachments/assets/7e027934-c776-440c-8d12-ff6f66e49be9) |

If we want to keep the combination of small thumbnailDisplay + margin 0 to accomodate better the thumbnail for now, we will also need to set the border radius of top-left and bottom-left corners to 0.

![image](https://github.com/user-attachments/assets/40132c28-c480-47c7-8127-a820ed5431cc)

Reply:
@AlexVelezLl I'd love to apply these updates (where appropriate) to KCard but I hesitated to get into changing KDS if I could avoid it for now.

That said, @MisRob if you could give a look here to some of the /deep/ styles I used that'd be really great.

The issue I ran into was that when I used `small` then the footer extended to the right, underneath the thumbnail. We want the footer to stop at the thumbnail (ie, it's the same width as title & description) -- but also for the thumbnail to be smaller.

The changes I've used kind of resize everything to accommodate the specs.

Reply:
I've updated the border-radius, great catch thanks Alex!

Reply:
> The issue I ran into was that when I used small then the footer extended to the right, underneath the thumbnail. We want the footer to stop at the thumbnail (ie, it's the same width as title & description) -- but also for the thumbnail to be smaller.

This sounds to me as possible inconsistency between designs and `KCard` implementation, or between the many designs, we may need to talk about. Let's chat on Slack so I can understand this better first.